### PR TITLE
test: Adjust special cases for ubuntu-stable moving to 20.10

### PR DESCRIPTION
**Pull request by martinpitt**
See https://github.com/cockpit-project/bots/pull/1135

- [ ] Land in lockstep with https://github.com/cockpit-project/bots/pull/1135

### CODE REVIEW

**Comment by martinpitt** [comment]
Apparently the [libvirt default network](https://logs-https-cockpit.apps.ci.centos.org/logs/pull-14457-20200807-062544-74692d9c-ubuntu-stable-bots-1135/log.html#266-2) is broken.

**Comment by martinpitt** [comment]
Almost! Forgot the hack in [TestKerberos](https://logs-https-cockpit.apps.ci.centos.org/logs/pull-14457-20200807-082123-0b43a25b-ubuntu-stable-bots-1135/log.html#297-2), otherwise this looks good.

File: test/verify/check-docker [hunk 1]
```diff
@@ -25,7 +25,7 @@ from testlib import *
 @skipPackage("cockpit-docker")
 @skipImage("No docker packaged", "rhel-8-3", "rhel-8-3-distropkg", "centos-8-stream")
 @skipImage("Not supporting cockpit-docker on CoreOS", "fedora-coreos")
-@skipImage("Not supporting cockpit-docker on Debian and Ubuntu >= 20.04", "debian-stable", "debian-testing", "ubuntu-2004")
+@skipImage("Not supporting cockpit-docker on Debian and Ubuntu >= 20.04", "debian-stable", "debian-testing", "ubuntu-2004", "ubuntu-stable")
```
Comment (marusak) [inline test/verify/check-docker:28]: I think we now only test docker on centos-7?

Comment (martinpitt) [reply test/verify/check-docker:28]: Heh, indeed -- and we don't test master on centos-7 any more. I thought that we still tested it on fedora-31, as we still ship it there. We need to add a hack to the tests to boot the image with cgroupsv1, and test it there. But oh well, won't be for long any more..

**Pull request merged**